pop-api/src/v0/assets/fungibles.rs

```diff
@@ -58,6 +58,65 @@ mod constants {
 	pub(super) const BURN: u8 = 20;
 }
 
+pub mod events {
```

Comment:
Missing doc comment for the public module. What is the point of this module? Some documentation will help users know why these events exist.

Reply:
I also think that the usage of these events should be demonstrated somewhere in this PR - e.g. unit or integration tests.

Reply:
This will take me 2 hours. Will push something later tonight

Reply:
EDIT: will do this tomorrow, created a separate issue for it to not delay this getting merged #181

Reply:
As for docs, see commit `docs: events contract`